README.md

```diff
@@ -1,37 +1,79 @@
+The `CageConfigurator` is a user interface to create a config file including an icon to be displayed on the secure desktop, the program which should be started and an optionally additional program. The additional program can be choosen out of a list of "trustworthy" applications.
+ As soon as the config file has been saved, a link to the config which is stored under `C:\Users\Public\Documents\SharkCage\` will be saved in the Registry under the following path: `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\SharkCage\Configs`.
+ The config contains json data and is protected using ACL's to be read and edited only by the `CageConfigurator` in a secure environment, the `CageService` as well as the administrator group.
+
+<details><summary markdown="span"><code>Fold out to see sample</code></summary>
+
+<img width="1680" alt="Screenshot CageConfigurator" src="https://user-images.githubusercontent.com/1786772/43733731-e88c2e50-99b5-11e8-84a5-860351997f55.png">
+
+</details>
+
+### SharkCageInstaller
+
+The `SharkCageInstaller` is used to install all project applications, starts the `CageService` and sets some keys in the registry.
 
 ## Installation
 
-1. Build the project
-1. Create the folder `C:\sharkcage` and copy `CageService.exe`, `CageManager.exe` and `ImageSelectDialog.exe` to this folder.
-1. Open `cmd` or `powershell` as administrator and add `CageService.exe` to the services: `sc create "shark-cage-service" binPath="C:\sharkcage\CageService.exe" obj= "NT AUTHORITY\SYSTEM"`
-1. Configure and start the `CageService`: `sc config "shark-cage-service" start=auto | sc start "shark-cage-service"`
+1. The `SharkCageInstaller` is used, to install all programs for this project (`CageService`, `CageManager`, `CageChooser` and `CageConfigurator`) and is hosted on Github. Follow this link and download the latest version: [Github-Releases](https://github.com/SharkCagey/HTWG_shark_cage/releases).
+1. Out of security perspective before installing, please make sure that the `SharkCageInstaller` is signed using the certificate issued to the HTWG Konstanz with the following fingerprint: `ADBE74BD39789DD111815DE59C60D715143E4620`.
+1. Execute the installer and follow the instructions. For installing the service, the `SharkCageInstaller` needs to run with admininistration privileges. Please make sure that the "User Account Control"-dialog shows the HTWG Konstanz as the verified publisher.
+   <details><summary markdown="span"><code>Fold out to see sample</code></summary>
+
+   <img width="500" alt="Screenshot User Account Control dialog" src="https://user-images.githubusercontent.com/1786772/43678105-71343f90-980d-11e8-89f5-9a77c63b86fa.png">
 
-## Useful Links
+   </details>
 
-Our [Jenkins Server](http://35.162.112.109:8080/)
\ No newline at end of file
+## Building from source
+
+1. Clone or download this repository
+1. Build the project
+1. Start the installer and follow the instructions.
```

Comment:
why do we need to start the installer when building from source?

Reply:
It's the easiest way I think otherwise you have to create and copy the config by your own and have to set the registry key etc. Any objection?

Reply:
was thinking strictly about the build process. but you are right, using the installer is the easiest way to use the newly built files. 

maybe we should also add a little hint how to actually built the project then (point 2):
1. Build the project (`SharkCage.sln`, VS2017 required) with one of the available build targets (debug / release)

Reply:
With stock VS2017 it is not actually possible to build the project. *.vdproj template is no longer supported in VS2017 by default
[Migration Report.pdf](https://github.com/SharkCagey/HTWG_shark_cage/files/2267180/Migration.Report.pdf)

Reply:
@SailReal to your previous comment:

> It's the easiest way I think otherwise you have to create and copy the config by your own and have to set the registry key etc.

I think It would be nice, if that sweet post-build Powershell script running after launch of chooser in debug would do that... that would eliminate the need to use the installer in a dev environment. Opinions?

Reply:
> Do you want me to include the necessary changes to the script in this PR?

Would be awesome 😍!!!!